src/Header/Header.js

```diff
@@ -9,9 +9,31 @@ const Header = () => {
           RANCID TOMATILLOS
         </h1>
       </Link>
-      <Link to="/Favorites">
-        <h3>Favorites</h3>
-      </Link>
+      <div id="linkContainer">
+        <NavLink 
+          exact to="/"
+          activeStyle={{
+            fontWeight: "bolder",
+            textDecoration: "underline",
+            textDecorationColor: "#AB2300"
+          }}
+          >
+          <h3 
+          className="headerLink"
+          >Home
+          </h3>
+        </NavLink>
```

Comment:
Great use of nav links. I think this is going to make it very user friendly to see the style attributes.